pipeline/splicing/calcVarPriors.py

```diff
@@ -1235,16 +1548,94 @@ def getPriorProbAfterGreyZoneSNS(variant, boundaries):
     Checks that variant is after the grey zone and is a single nucleotide substitution
     Checks that variant is either a missense or nonsense mutation
     Returns a dictionary containing prior probability of pathogenecity and predicted qualitative enigma class
+    Dictionary contains other values that are set to either N/A, "-", or 0 because they are not relevant
```

Comment:
are all of these necessary? can they be reduced to a single value for consistency/simplicity?

Reply:
Again "N/A" and "-" refer to specific conditions where those values are not checked or are irrelevant

Reply:
gotcha